### DISCUSSION TITLE
Flash of unstyled content (FOUC) , css modules and global css.

### DISCUSSION THREAD

**Discussion by SergiiMost** [vercel/next.js#14412] [Help]
In my project I have a **global css file**,  and I'm using **css modules (*.module.css)**. I've noticed that there is a FOUC. Is there a way to fix it or should I just switch to another styling solution that doesn't have this problem?

**Comment by sync** [comment]
The issue has been reported multiple times main one is here: (https://github.com/vercel/next.js/issues/10557), and there is an open pr for it: https://github.com/vercel/next.js/pull/12843.

**Comment by stijnklomp** [comment]
This appears to still be an issue on the latest version. [Here](https://stackblitz.com/edit/vercel-next-js-clzexc?file=app%2FPageDynamicContent.tsx) is an example project that shows the result, taking from [this](https://github.com/vercel/next.js/issues/44994#issuecomment-1404294851) post.

**Comment by najeebputhiyallam** [comment]
I have same issue here https://next.najeebdigital.xyz/ when i navigate to about or any other page, css module load later creating a jumping effect to layout.

**Comment by zdettwiler** [comment]
Likewise, I have the same issue. I have the impression it's rather recent...
